lume.lua

```diff
@@ -707,9 +711,11 @@ function lume.hotswap(modname)
   xpcall(function()
     package.loaded[modname] = nil
     local newmod = require(modname)
-    if type(oldmod) == "table" then update(oldmod, newmod) end
+    if type(newmod) == "table" and type(oldmod) == "table" then
```

Comment:
Only merge if both oldmod/newmod are both tables. This could be replaced with a check inside update but since the other call paths need to check out of update I thought consistency would be better.